pkg/dns/dns_test.go

```diff
@@ -427,6 +429,7 @@ func TestNamedHeadlessServiceEndpointUpdate(t *testing.T) {
 
 func TestNamedHeadlessServiceEndpointDelete(t *testing.T) {
```

Comment:
Add a testcase for a en endpoint that requests reverse DNS but the pod doesn't confirm?